docs/references/cli-arguments.md

```diff
@@ -37,6 +37,9 @@ Following table describes all the flags that are available:
 | --alsologtostderr                    |`boolean`  | `false`                         | Logs are written to standard error as well as to files.|
 | --anonymous-reports                  |`string`   | `true`                          | Send anonymized usage data to help improve Kong.|
 | --apiserver-host                     |`string`   | none                            | The address of the Kubernetes Apiserver to connect to in the format of protocol://address:port, e.g., "http://localhost:8080. If not specified, the assumption is that the binary runs inside a Kubernetes cluster and local discovery is attempted.|
+| --disable-ingress-extensionsv1beta1  |`boolean`  | `false`                         | Disable processing Ingress resources with apiVersion `extensions/v1beta1`.|
```

Comment:
non-blocking: I don't see a point in maintaining this table separately from pflag flag definitions in the code (and the `--help` resulting from there).

By maintaining it, we're asking for extra maintenance burden, and discrepancy bugs.

Tracked (and the necessity of this table to be discussed with @rainest @hbagdi) in #880